index.js

```diff
@@ -61,6 +68,13 @@ exports.register = function(server, options, next) {
         if (request.query.json === '1') {
           return reply(combinedData).type('application/json');
         }
+
+        if (typeof options.preResponse === 'string') {
+          return str2fn(server.methods, options.preResponse)(request, options, combinedData, reply, () => {
+            reply.view(viewConfig.view, combinedData);
```

Comment:
If you need to override you just call reply in your method and ignore the next param so the reply.view isn't called.